<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EqItem</class>
 <widget class="QWidget" name="EqItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>31</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>501</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="type">
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Parametric</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bass Shelf</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Treble Shelf</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Phase Changer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lowpass LR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Highpass LR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lowpass BW</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Highpass BW</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bandpass</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bypass</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="Order">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="spinBox_fc">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>20000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="spinBox_gain">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="spinBox_q">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.990000000000009</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="selectedBtn">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
